Golang Mod Analyzer fails on modules that have no dependencies

**Describe the bug**
The experimental Golang Mod Analyzer fails on modules that have no dependencies, such as https://github.com/hashicorp/golang-lru.

**Version of dependency-check used**
Dependency-Check Core version 5.3.1 (cli)

**Log file**
```
WARN  - An error occurred while analyzing '/home/user/Repositories/golang-lru/go.mod' (Golang Mod Analyzer).
2020-03-13 11:47:07,211 org.owasp.dependencycheck.AnalysisTask:91
DEBUG - 
org.owasp.dependencycheck.analyzer.exception.AnalysisException: JSON not exactly matching output of `go mod edit -json`
        at org.owasp.dependencycheck.data.golang.GoModJsonParser.process(GoModJsonParser.java:99)
        at org.owasp.dependencycheck.analyzer.GolangModAnalyzer.analyzeDependency(GolangModAnalyzer.java:277)
        at org.owasp.dependencycheck.analyzer.AbstractAnalyzer.analyze(AbstractAnalyzer.java:131)
        at org.owasp.dependencycheck.AnalysisTask.call(AnalysisTask.java:88)
        at org.owasp.dependencycheck.AnalysisTask.call(AnalysisTask.java:37)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassCastException: class javax.json.JsonValue$1 cannot be cast to class javax.json.JsonArray (javax.json.JsonValue$1 and javax.json.JsonArray are in unnamed module of loader 'app')
        at org.glassfish.json.JsonObjectBuilderImpl$JsonObjectImpl.getJsonArray(JsonObjectBuilderImpl.java:179)
        at org.owasp.dependencycheck.data.golang.GoModJsonParser.process(GoModJsonParser.java:82)
        ... 8 common frames omitted
```

**To Reproduce**
Run `dependency-check --enableExperimental --scan .` in the https://github.com/hashicorp/golang-lru repo.

**Expected behavior**
The Golang Mod Analyzer should run successfully.

**Additional context**
While a failure in a module with no dependencies is not a huge problem in itself, this is causing lots of unnecessary errors in larger projects.

--- a/core/src/main/java/org/owasp/dependencycheck/data/golang/GoModJsonParser.java
+++ b/core/src/main/java/org/owasp/dependencycheck/data/golang/GoModJsonParser.java
@@ -77,7 +77,7 @@ public class GoModJsonParser {
         LOGGER.debug("Beginning go.mod processing");
         try {
             final JsonObject composer = jsonReader.readObject();
-            if (composer.containsKey("Require")) {
+            if (composer.containsKey("Require") && !composer.isNull("Require")) {
                 LOGGER.debug("Found modules");
                 final JsonArray modules = composer.getJsonArray("Require");
                 for (JsonObject module : modules.getValuesAs(JsonObject.class)) {

